docs/contribute/development/rust/coding_guidelines.rst

```diff
@@ -17,6 +17,104 @@ Writing Rust Code
 
 .. document:: Coding Guidelines Rust
    :id: doc__rust_coding_guidelines
-   :status: invalid
+   :status: valid
 
-.. note:: The Rust coding guidelines are not yet defined. This is a placeholder for future content.
+
+Coding Guidelines
+=================
+
+The Safety-Critical Rust Consortium aims to make Rust suitable for use in
+automotive and other safety-critical environments by building and maintaining a
+set of essential tools that are vetted by the community for certification
+purposes. They track the development status of these tools and document their
+progress. The consortium is considering whether to develop specialized training
+materials for safety-critical Rust, though this may require a separate group.
+Their current activities include supporting a qualified compiler (with
+Ferrocene available for some targets), developing a certified core library,
+working on tools for coding style verification, and assessing the need for
+static analysis and code metrics tools. Some tools, such as MC/DC coverage
+reporting and code metrics generators, are still unavailable, and the
+consortium is evaluating what further tooling and support are necessary to
+enable certification and safe use of Rust in automotive applications.
+
+`Safety-Critical Rust Consortium <https://rustfoundation.org/safety-critical-rust-consortium>`_
+
+`Safety-Critical Rust Consortium Guidelines <https://github.com/rustfoundation/safety-critical-rust-consortium/tree/main/subcommittee/coding-guidelines/>`_
+
+`Learn unsafe Rust <https://google.github.io/learn_unsafe_rust/>`_
+
+`Rust language <https://doc.rust-lang.org/book/ch20-01-unsafe-rust.html>`_
```

Comment:
Imo these references are - while worth reading - misleading, as unsafe Rust does not necessarily mean bad code, nor does sticking to safe Rust mean perfect code for safe applications.